addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -214,6 +216,28 @@ def _update_accrual(self):
 
             holiday.write(values)
 
+    def _compute_accrual_plan(self):
+        accrual_plans=self.env['hr.accrual.plan'].search([("company_id","=",self.env.company.id)])
+        for record in self:
```

Comment:
It's better to be explicit and explain on what kind of record you work.
```suggestion
        for allocation in self:
```